### PR TITLE
Handle a trace where 'action' is null

### PR DESCRIPTION
**Pull request by YordanPavlov**
A fix for a case where ETH node returns trace containing
```json
{
  "action": null
}
```

this has happened in Erigon v2.33.0

### CODE REVIEW

File: blockchains/eth/eth_worker.js [hunk 1]
```diff
@@ -23,20 +23,22 @@ class ETHWorker extends BaseWorker {
     this.feesDecoder = new FeesDecoder(this.web3, this.web3Wrapper);
   }
 
+  parseEthInternalTrx(result) {
```
Comment (YordanPavlov) [inline blockchains/eth/eth_worker.js:26]: Move as a separate function to facilitate testing

File: blockchains/eth/lib/filter_errors.js [hunk 2]
```diff
@@ -45,7 +45,8 @@ function setErrors(traces) {
  }
 */
 function filterNonActions(traces) {
-  return traces.filter(trace => typeof trace.action !== 'undefined');
+  // Leave only 'truthy' action values
```
Comment (YordanPavlov) [inline blockchains/eth/lib/filter_errors.js:48]: This handles both 'undefined' and `null`

File: docker/Dockerfile [hunk 3]
```diff
@@ -37,7 +37,7 @@ COPY package.json AUTHORS LICENSE index.js /opt/app/
 COPY blockchains /opt/app/blockchains
 COPY lib/ /opt/app/lib
 COPY test /opt/app/test
-COPY docker/wait_for_services.sh /opt/app/docker
+COPY docker/wait_for_services.sh /opt/app/docker/wait_for_services.sh
```
Comment (YordanPavlov) [inline docker/Dockerfile:40]: An unrelated fix, this file was not properly copied.

File: docker/docker-compose.yaml [hunk 4]
```diff
@@ -26,6 +26,8 @@ services:
       - /var/run/docker.sock:/var/run/docker.sock
     logging:
       driver: none
+    depends_on:
```
Comment (YordanPavlov) [inline docker/docker-compose.yaml:29]: An unrelated fix, Kafka should wait for Zookeeper.

**Review by kalkov — APPROVED** [review]

**Review by spiderjako — APPROVED** [review]
